docs/supported_formats.md

```
@@ -63,35 +63,37 @@ datamodel-code-generator automatically detects the JSON Schema version:
 
 ### Feature Compatibility Matrix
 
-| Feature | OAS 3.0 | OAS 3.1 |
-|---------|---------|---------|
-| **Schema Base** | JSON Schema Draft 5 (subset) | JSON Schema 2020-12 (full) |
-| **Definitions Path** | `#/components/schemas` | `#/components/schemas` |
+| Feature | OAS 3.0 | OAS 3.1 | OAS 3.2 |
+|---------|---------|---------|---------|
+| **Schema Base** | JSON Schema Draft 5 (subset) | JSON Schema 2020-12 (full) | JSON Schema 2020-12 (full) |
+| **Definitions Path** | `#/components/schemas` | `#/components/schemas` | `#/components/schemas` |
 | **Nullable** |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 4; Actual: 1; Too few cells, row will be missing data

(MD056, table-column-count)